src/platform/plugins/private/files_management/public/components/diagnostics_flyout.tsx

```diff
@@ -42,7 +42,12 @@ export const DiagnosticsFlyout: FunctionComponent<Props> = ({ onClose }) => {
   });
 
   return (
-    <EuiFlyout ownFocus onClose={onClose} size="s">
+    <EuiFlyout
+      ownFocus
+      onClose={onClose}
+      size="s"
+      aria-label={i18nTexts.diagnosticsFlyoutAriaLabel}
```

Comment:
I prefer to use common patter with setting `aria-labelledby` attribute.   No need to add extra `flyout` word